object/token_oauth.go

```diff
@@ -328,10 +328,17 @@ func RefreshToken(grantType string, refreshToken string, scope string, clientId
 	}
 
 	// generate a new token
-	user, err := getUser(application.Organization, token.User)
+	organization := application.Organization
```

Comment:
I think we may need a more thorough fix for all such cases - for cross-org users

So define a function like `getUserFromApplication(application *Application)` and use it in all places

Reply:
@hsluoyz That make sense. However, I am not that familiar with the code base to do that big refactoring. Also, it will make PR big and difficult to review for you like previous one. 

This is a very critical flow that is broken right now causing user logouts on token refresh as this API is failing. I can take a look at making the separate PR that fixes this issue from other places if that works for you.

Reply:
@dacongda

Reply:
Also, I am not sure what you mean by `getUserFromApplication(application *Application)` here. Problem is that user's organization should be assumed to be same as application's organization. In refresh token API, we have user's information from refresh token. I am not sure how we are going to solve this problem by creating a new function `getUserFromApplication(application *Application)`.